Gemfile

```diff
@@ -11,7 +10,7 @@ gem 'rails', '3.2.22.2'
 
 # Database
 # gem 'sqlite3-ruby', :require => 'sqlite3'
-gem 'mysql2', '0.3.10'
+gem 'mysql2', '0.3.20'
```

Comment:
We have a pull request for it but it has yet to be merged.

https://github.com/otwcode/otwarchive/pull/2292